app/models/user.rb

```diff
@@ -73,6 +75,13 @@ def any_admin?
     admin? || beach_admin?
   end
 
+  def reporter_rank
```

Comment:
I'm going to refactor this.
This is too slow and only calculates the first 100 positions.

We need to:
1 - Calculate it only for active users (when doing it, make sure all users have a value)
2 - Calculate it every X minutes
3 - It would be nice to put the old statuses (older than a month/week) in a historic table, to make this faster)

Reply:
I thought we were only interested in the top100:)